vitrina/datasets/search_indexes.py

```diff
@@ -35,6 +36,10 @@ def prepare_category(self, obj):
             categories.append(obj.category.pk)
         return categories
 
+    def prepare_organization(self, obj):
+        if obj.organization.pk != obj.management_area:
```

Comment:
I think `obj.management_area` is a method and should be called with `()`?

Not sure if this is related with this code snippet, but when I tested it, I has in total 1099 datasets, but when I added all `/dataset/stats/supervisor/` numbers, I got 941 datasets.  Where are the rest 158 datasets?

Reply:
The total number of datasets that you calculated will never be equal to all datasets shown in search, because as stated in the task we only show jurisdictions with more than 1 child and if a root organization has only 1 child, it will not be shown.

But you are right, some organizations were being hidden by elasticsearch, because it defaults to 10 facets. Fixed this and now it shows all root organizations that have more than one child.

Reply:
Also while yes, obj.management_area (now changed to jurisdiction) is a method, when I call it with parentheses it breaks the filtering.